### ISSUE TITLE
Recipe conflicts

### ISSUE THREAD

**Issue by Juuxel** [AK9Official/All-of-Fabric#32]
- 2x2 square of bamboo:
  - Bamboo Planks (Blockus)
  - Bamboo Block (Bamboo Tweaks)
- 3x2 rectangle of various Terrestria planks:
  - Trapdoors (Terrestria)
  - Walls (Extra Pieces)
    - This one is possibly a bug with the Terrestria piecepack. AFAIK wooden walls shouldn't have a regular crafting recipe.

**Comment by Juuxel** [comment]
Yep, the wall issue is a problem with the piecepack. The vanilla piecepack sets wooden walls as uncraftable:
https://github.com/AllOfFabric/AOF/blob/83b6bd5612d57717794004dd610ef31d53635523/.minecraft/config/extrapieces/piecepacks/default.json#L479-L490

The `uncraftable` array is missing from the Terrestria piecepack JSON.

**Comment by coderbot16** [comment]
If possible I would like to ship this change in a Terrestria update, because it will affect everyone else too! At some point I need to migrate over to the new config system, and I'll fix this stuff up there.
